lib/jekyll/converters/markdown.rb

```diff
@@ -5,8 +5,7 @@ class Markdown
 
       PARSERS = { 'redcarpet' => RedcarpetParser,
                   'kramdown' => KramdownParser,
-                  'rdiscount' => RDiscountParser,
-                  'maruku' => MarukuParser }
+                  'rdiscount' => RDiscountParser}
```

Comment:
Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.